### PR TITLE
Store/retrieve app state all in one place (fixes #347)

### LINKED ISSUE #347: Consolidate application error state
Using two different reducers, one at `reducers/app` and one at `reducers/management`, we store application state errors in both places (grep for `actionTypes.APP_ERROR`). We only need to store that in one place, probably just in `reducers/app`. The management app can access `state.app` to get the error.

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/payments-ui/issues/347

### CODE REVIEW

File: public/js/reducers/management.js [hunk 1]
```diff
@@ -21,12 +20,6 @@ export default function management(state, action) {
         });
       }
       return state;
-    case actionTypes.APP_ERROR:
-      return Object.assign({}, initialMgmtState, {
-        error: {
-          debugMessage: action.error.debugMessage,
-        },
-      });
```
Comment (kumar303) [inline public/js/reducers/management.js]: huh, I guess this wasn't tested

**Comment by muffinresearch** [comment]
This is going to clash with the redux 2.0 upgrade patch. :/

**Comment by kumar303** [comment]
I'll fix this up after https://github.com/mozilla/payments-ui/pull/392

**Comment by muffinresearch** [comment]
r+wpt

**Pull request merged**